src/components/admin/MainIssueController.jsx

```diff
@@ -146,7 +147,7 @@ export default class MainIssueController extends FalcorController {
             window.alert(`${article.title} has no authors. Please correct this`);
             return false;
           }
-          if (/http(?!s)/.test(article.html)) {
+          if (!hasHttpsURL(article.html)) { // if the function returns true, the url is correct
```

Comment:
I like the general name of the function, but I don't think the semantics work I'm afraid :'(.

This sounds like it returns true if there's a https URL in it, and false if not, when in this case it should return true also if there aren't any urls at all.

What about `if (hasNonHttpsURL(article.html)`?